.github/workflows/update-sdk.yaml

```diff
@@ -15,18 +15,23 @@ permissions:
 
 jobs:
   update-version:
-    # Disable auto PR for now
-    if: false
     runs-on: buildjet-4vcpu-ubuntu-2204
+    strategy:
+      matrix:
+        tag: ['latest'] # Currently only "latest" tag is available across all packages
 
     steps:
       - uses: actions/checkout@v3
       - uses: FuelLabs/github-actions/setups/node@master
 
-      - name: Check versions
-        id: check_versions
-        run: |
-          cd packages/update-sdk
-          pnpm check-versions
+      - name: Checking updates
+        uses: FuelLabs/github-actions/update-sdk@hf/feat/update-sdk
```

Comment:
We should merge the action first before we merge this?

Reply:
@luizstacio exactly, can I have a review here?
https://github.com/FuelLabs/github-actions/pull/26

Reply:
https://github.com/FuelLabs/fuels-wallet/pull/1143/commits/ec24de76534e6f6dfe2b5831a3c511b13c8c00fd